packages/adapters/gatewallet/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { isInMobileBrowser } from '@tronweb3/tronwallet-abstract-adapter';
+
+export function supportGateWallet() {
+    return !!(window.gatewallet && window.gatewallet.tronLink);
```

Comment:
This function worked in mobile app but it seems strange. The first time I visit the website, I can not get `window.gatewallet.tronLink`. But after i click refresh button on the top right once or more, the `window.gatewallet.tronLink` will be valid. Is this a normal case ?

Reply:
Yes. it's my application problem. not adapter.  i'll check it.